util/i2csvg/README.md

```diff
@@ -0,0 +1,159 @@
+# i2csvg -- Generate svg pictures of i2c commands from text description
+
+i2csvg is a Python tool to read i2c transactions from a text file and
```

Comment:
usually you have `I2C` but here and above you have `i2c`, probably want to standardize on `I2C`?